tests/no_cycles.rb

```diff
@@ -0,0 +1,50 @@
+require 'find'
```

Comment:
Do you need `require_relative '../lib/color'` since you colorized the last line?

Reply:
Well, `../lib/color` is required by the `test_all` script, so it isn't necessary here. But I'm not sure if I should rely on that. I guess it's better to be sure, as the `test_all` script may change in the future.

Reply:
Added.